scripts/irods/test/test_igroupadmin.py

```diff
@@ -72,6 +71,63 @@ def tearDownClass(self):
             self.admin.__exit__()
             admin_session.assert_icommand(['iadmin', 'rmuser', self.admin.username])
 
+    def test_create_user_with_password_in_igroupadmin__6887(self):
+        USERS = { 'alice':42,
```

Comment:
Python convention questions ...

Is the use of capital letters here a convention in Python?
Why not use lowercase letters? _(I'm guessing it's supposed to provide signal of being treated as readonly?)_

Reply:
It doesn't have the strength it does in e.g. Ruby, where a warning is invoked if you change it, but yes.  I think the convention across languages was to have constants announced as all-caps, esp in scripting languages where the "const" is less enforcible.